tests/default_vars.sh

```diff
@@ -370,7 +370,7 @@ export OUTPUT_GRID="'cubed_sphere_grid'"
 export OUTPUT_FILE="'netcdf'"
 export ZSTANDARD_LEVEL=0
 export IDEFLATE=0
-export NBITS=0
+export QUANTIZE_NSD=0
```

Comment:
Since we will not use NBITS anymore, would you please create an issue on global-workflow to let them know this update?

Reply:
Opened NOAA-EMC/global-workflow#2053